db/text/en/policies.xml

```diff
@@ -0,0 +1,97 @@
+<db id="policies">
+    <!-- ============================ -->
+    <!-- Campaigns -->
+    <!-- ============================ -->
+    <!-- The tag here represents where the policy button will be displayed in the policy window-->
+    <campaign id="renewables" tag="env">
+        <!-- The title that will be displayed at the top of the policy -->
+        <name>Renewable energy information campaign</name>
+        <!-- The description text that will be displayed in the policy popup -->
+        <text>Start Information Campaign informing on the advantages of Renewable Energy.
+            Increases acceptance of renewable energy and their related policies by n %.
+        </text>
+        <!-- The effect the campaign will have on the probability of passing the env policies -->
+        <effect tag="env" value="0.2"/>
+        <!-- Number of turns that the campaign will take -->
+        <length value="1"/>
+    </campaign>
+
+    <campaign id="home" tag="demand">
+        <name>Home insulation</name>
+        <text>Start information campaign explaining the advantages of home insulation.
+            Increases acceptance of home insulation and their related policies by n %.
+        </text>
+        <effect tag="demand" value="0.2"/>
+        <length value="1"/>
+    </campaign>
+
+    <!-- ============================ -->
+    <!-- Policies -->
+    <!-- ============================ -->
+
+    <policy id="rooftop_PV" tag="env">
+        <name>Rooftop solar PV regulation</name>
+        <text>Mandatory rooftop solar PV installation by [year].
+            Increase of decentral production of Solar Energy by n %.
+        </text>
+        <!-- A requirement on the amount of support needed to attempt the policy -->
+        <requirement field="support" value="50" bound="min"/>
```

Comment:
This should actually be a base support that will impact the probability of passing the policy. So below this amount, the difference will reduce the probability of passing, above this is will increase the probability.

Reply:
We need to figure a proper way to reduce this, e.g.
```
probability = baseProbability - (baseProbability * (requirement - support))
```